force-app/main/default/classes/OrderTriggerHandler.cls

```diff
@@ -0,0 +1,109 @@
+public with sharing class OrderTriggerHandler {
+    public static void createAssetRecord(List<order> orderList, Map<Id, Order> oldOrderMap){
+        Set<ID> orderIdSet = new set<Id>();
+        for(Order order : orderList){
+            if(oldOrderMap.containskey(order.Id) && oldOrderMap.get(order.Id).Status == 'Draft' && order.Status=='Activated'){
+                orderIdSet.add(order.Id);
+            }
+        }
+        BusinessHours  bussinessHours = [SELECT Id, Name FROM BusinessHours Where Name='Test Bussiness Hours' limit 1]; 
+        List<OrderItem> orderItemList = [select id, Product2.Name, Product2.ProductCode , Order.AccountId, Order.OpportunityId, Order.CreatedDate,Product2.POC__c, UnitPrice, Quantity from OrderItem Where OrderId IN :orderIdSet AND Product2.Family='Software'];
```

Comment:
always use all capital cases for SOQL keywords (i.e. SELECT, FROM, WHERE, ORDER BY, LIKE etc.)